### ISSUE TITLE
Incorrect loss used

### ISSUE THREAD

**Issue by vladhondru25** [Tianxiaomo/pytorch-YOLOv4#397]
Hello,

After carefully checking the code, I can clearly see that the loss used is not Complete Intersection over Union (CIoU), as stated in the Yolo v4 paper, but rather regression is used for box prediction.

https://github.com/Tianxiaomo/pytorch-YOLOv4/blob/4ccef0ec8fe984e059378813e33b3740929e0c19/train.py#L265-L274

Can you confirm that's the case?

Thank you!
